### DISCUSSION TITLE
header inside nav, why?

### DISCUSSION THREAD

**Discussion by Mohammadtahirlashari** [heroui-inc/heroui#1986] [General]
header inside nav, why?

I have studied in MDN Docs. 

When we use <header> direct in <body>, it's role is as site banner landmark and it's recommended because assistive technologies.

Whn we use <header> inside any other html element like <main> <nav> <article> <section>, etc. it not behaves like banner.

My Question:

In NextUI, You have put header inside nav element. it means, if we use navbar form NextUI, then, assistive technologies fails to determin the site landmark. which is not good for web accessibility.

Please, correct me, If I'm wrong or convince me this is the best practice to keep header inside nav or keep nav inside header.

thank you

**Comment by carlosyan1807** [comment]
Yes, I also think they got `nav` and `header` in wrong order.
